fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -1008,6 +1009,9 @@ public void cancel() {
         if (coordRef != null) {
             coordRef.cancel();
         }
+        if (mysqlLoadId != null) {
```

Comment:
Should we reset `mysqlLoadId` after job is cancelled or finished?

Reply:
The next Stmt  in the same session will create a new `ExecuteStmt` object in class `ConnectProcessor`.
So this load id will gc after load finished and it's the reson why coordRef is not assgin null after call the cancel.

Reply:
And in mysql Load manager, I also had check the loadId exist before use it.